index.html

```diff
@@ -0,0 +1,62 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>einstein</title>
+</head>
+<main id="main">
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Albert Einstein</h1>
+    <p>Pioneer Scientist in the field of physics</p>
+    <figure id="img-div">
+      <img
+        id="image"
+        src="einstein.jpg"
+        alt="einstein"
+      />
+    </figure>
+    <section>
+        <h2>About</h2>
+      <p id="tribute-info">
+        Albert Einstein was born in Ulm, in the Kingdom of Württemberg in the
+        German Empire, on 14 March 1879 into a family of secular Ashkenazi
+        Jews.His parents were Hermann Einstein, a salesman and engineer, and
+        Pauline Koch. In 1880, the family moved to Munich, where Einstein's
+        father and his uncle Jakob founded Elektrotechnische Fabrik J.
+        Einstein & Cie, a company that manufactured electrical equipment based
+        on direct current. Albert attended a Catholic elementary school in
+        Munich, from the age of five, for three years. At the age of eight, he
+        was transferred to the Luitpold Gymnasium (now known as the Albert
+        Einstein Gymnasium), where he received advanced primary and secondary
+        school education until he left the German Empire seven years later. In
+        1894, Hermann and Jakob's company lost a bid to supply the city of
+        Munich with electrical lighting because they lacked the capital to
+        convert their equipment from the direct current (DC) standard to the
+        more efficient alternating current (AC) standard.The loss forced the
+        sale of the Munich factory. In search of business, the Einstein family
+        moved to Italy, first to Milan and a few months later to Pavia. When
+        the family moved to Pavia, Einstein, then 15, stayed in Munich to
+        finish his studies at the Luitpold Gymnasium. His father intended for
+        him to pursue electrical engineering, but Einstein clashed with the
+        authorities and resented the school's regimen and teaching method. He
+        later wrote that the spirit of learning and creative thought was lost
+        in strict rote learning. At the end of December 1894, he traveled to
+        Italy to join his family in Pavia, convincing the school to let him go
+        by using a doctor's note.During his time in Italy he wrote a short
+        essay with the title "On the Investigation of the State of the Ether
+        in a Magnetic Field". He was awarded the Nobel Prize in Physics in
+        1901.
+      </p>
+      <a
+        href="https://en.wikipedia.org/wiki/Albert_Einstein"
+        id="tribute-link"
+      ></a>
```

Comment:
Great use of semantic tags :heavy_check_mark:  kindly use a div tag to make the website be easily interpreted :100:

Reply:
Good job appreciating the use of semantic tags here 👍